icechunk/default.yaml

```diff
@@ -0,0 +1,31 @@
+repos:
```

Comment:
can we move this file src/bin/icechunk?

Reply:
I have now removed the file altogether (https://github.com/earth-mover/icechunk/pull/716/commits/5b03fbd9d6c39eedbd11ffaad4f3db4c0f305a76).

Should we instead decide on a standard location for the config, not relative to the repo? How about `$HOME/.config/icechunk/cli_config.yaml`, obtained through [dirs::config_dir()](https://crates.io/crates/dirs)?

Reply:
love it

Reply:
Did that here https://github.com/earth-mover/icechunk/pull/716/commits/8f8d89fd1e97f79aa226a938cc7348495bec0bdb

On Mac, this actually [defaults to](https://docs.rs/dirs/latest/dirs/fn.config_dir.html) `$HOME/Library/Application Support`.